### PR TITLE
Added profile Signal US Core Service Request

### PR DESCRIPTION
**Pull request by arunsundar-m**
Added profile Signal US Core Service Request
Bound Service Type from HealthCare Service profile to category:us-core slice

### CODE REVIEW

File: input/resources/StructureDefinition-profile-SignalUSCoreServiceRequest.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      {
+        "id": "ServiceRequest.intent",
+        "path": "ServiceRequest.intent",
+        "binding": {
+          "strength": "required",
+          "valueSet": "http://hl7.org/fhir/ValueSet/request-intent"
+        }
+      },
+      {
+        "id": "ServiceRequest.category:us-core",
```
Comment (enjoysparkling) [inline input/resources/StructureDefinition-profile-SignalUSCoreServiceRequest.json:32]: Change back US Core to it's default definition.  Don't change this ValueSet, define a new slice under ServiceRequest.category, I would suggest `ServiceRequest.category:serviceCategory`

**Review by enjoysparkling — CHANGES_REQUESTED** [review]
see in-line notes - need to revert change on us-core and create new slice

File: input/resources/StructureDefinition-profile-SignalUSCoreServiceRequest.json (hunk 1 above)
Comment (arunsundar-m) [reply input/resources/StructureDefinition-profile-SignalUSCoreServiceRequest.json]: @enjoysparkling Please review the changes https://github.com/enjoysparkling/signal-mso-fhir-profiles/pull/84/commits/7ff59fdbf25fb0ab57b0fd22d5a94a14ff3ab57d

File: input/resources/StructureDefinition-profile-SignalUSCoreServiceRequest.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "signal-profile-us-core-service-request",
+  "url": "https://signalbhn.org/fhir/StructureDefinition/StructureDefinition-profile-SignalUSCoreServiceRequest",
+  "name": "StructureDefinition-profile-SignalUSCoreServiceRequest",
```
Comment (enjoysparkling) [inline input/resources/StructureDefinition-profile-SignalUSCoreServiceRequest.json:5]: Need to change this:  
```
Message  : Improper name: 'StructureDefinition-profile-SignalUSCoreServiceRequest'. Name should be usable as an identifier for the module by machine processing applications such as code generation (capital letter followed by any number of letters, digits or underscores with a maximum of 255 characters).
Project  : Signal MSO FHIR Profiles
Resource : StructureDefinition-profile-SignalUSCoreServiceRequest
Id       : sdf-0
Severity : Warning
Source   : StructureDefinition-profile-SignalUSCoreServiceRequest
Context  : name
```

**Review by enjoysparkling — CHANGES_REQUESTED** [review]
Changes: 
- Change the .url element to be https://signalbhn.org/fhir/StructureDefinition/servicerequest- I would like to standardize all resources like this
- Change .id element to be signal-servicerequest
- Change .name element to be signal-servicerequest-profile - I would like to standardize all resources like this
 Add the .title element to be ServiceRequest Profile

File: input/resources/StructureDefinition-profile-SignalUSCoreServiceRequest.json (hunk 2 above)
Comment (arunsundar-m) [reply input/resources/StructureDefinition-profile-SignalUSCoreServiceRequest.json]: @enjoysparkling I have made the requested changes for id, name and URL elements.

#e8d56dabc9e357bbbeb04bff77e39bc121acc4c7

Comment (arunsundar-m) [reply input/resources/StructureDefinition-profile-SignalUSCoreServiceRequest.json]: Do we need to do the same for Signal US Core Organization profile and Signal Health Care Service profiles?
@enjoysparkling

**Review by enjoysparkling — APPROVED** [review]

**Pull request merged**